dist/compiler/extensions/generateJS.js

```diff
@@ -70,28 +67,29 @@ var generatorFns = {
   },
 
   open_HTML_ELEMENT: function open_HTML_ELEMENT(instruction, code) {
-    var state = instruction.state;
     var tdBody = instruction.tdBody;
     var elCount = instruction.elCount;
+    var attrIdx = instruction.attrIdx;
     var key = instruction.key;
     var namespace = instruction.namespace;
 
     namespace = namespace ? ", '" + namespace + "'" : "";
-    if (state !== STATES.HTML_ATTRIBUTE) {
+    if (attrIdx === null) {
       var fragment = "      var el" + elCount + " = td." + util.getTdMethodName("createElement") + "('" + key + "'" + namespace + ");\n";
       code.push(tdBody, { fragment: fragment });
     }
   },
   close_HTML_ELEMENT: function close_HTML_ELEMENT(instruction, code) {
-    var state = instruction.state;
     var parentNodeName = instruction.parentNodeName;
     var elCount = instruction.elCount;
     var tdBody = instruction.tdBody;
+    var attrIdx = instruction.attrIdx;
+    var node = instruction.node;
 
-    if (state === STATES.ESCAPABLE_RAW) {
-      var fragment = "      el" + (elCount - 1) + ".defaultValue += td." + util.getTdMethodName("nodeToString") + "(el" + elCount + ");\n";
+    if (node[1].escapableRaw) {
```

Comment:
`node[1].escapableRaw different` then `state === STATES.ESCAPABLE_RAW` because of `item.state = item.previousState;`